allan_variance.py

```diff
@@ -53,9 +53,15 @@ def allan_variance(x, dt=1, min_cluster_size=1, min_cluster_count='auto',
     tau : ndarray
         Averaging times for which Allan variance was computed, 1-d array.
     avar : ndarray
-        Values of AV. The 0-th dimension is the same as for `tau`. The trailing
-        dimensions match ones for `x`.
+        Values of AVAR. The 0-th dimension is the same as for `tau`. The
+        trailing dimensions match ones for `x`.
+
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Allan_variance
     """
+    if input_mode.lower() not in ("increment", "mean"):
```

Comment:
I'm basing on the codebase of scipy and scikit-learn, it's just established not to do that in such cases. Also when `input_mode` is not a string, it won't work as expected.